packages/appium/lib/cli/setup-command.js

```diff
@@ -0,0 +1,163 @@
+import _ from 'lodash';
+import { SETUP_SUBCOMMAND } from '../constants';
+import {runExtensionCommand} from './extension';
+import { fs } from '@appium/support';
+
+/**
+ * Driver names listed in KNOWN_DRIVERS to install by default.
+ */
+export const MOBILE_DRIVERS = ['uiautomator2', 'espresso', 'xcuitest'];
+export const SUBCOMMAND_MOBILE = 'mobile';
+
+
+/**
+ * Driver names listed in KNOWN_DRIVERS to install by default.
+ */
+export const DESKTOP_APP_DRIVERS = ['mac2'];
```

Comment:
how about windows?

Reply:
It is not in https://github.com/appium/appium/blob/e972d668fe77854b52bb219fee9aaaee0f15ca98/packages/appium/lib/constants.js#L40-L49 so i haven't added at this time

Reply:
Would the suggested host platform limitation also apply here, like for mobile? e.g. install mac2 only if running macOS